pkgs/applications/office/libreoffice/darwin/update-test.nix

```diff
@@ -0,0 +1,47 @@
+# run the tests with nixt <absolutePath to parent dir> -v
```

Comment:
I don't think we need this at all. We can simply locally revert a last update and see if the update script works. This also gives us a very very false sense of security because there website could easily change tomorrow and we would have green testing.

Reply:
I added an integration test for the website to solve the false sense of security issue.

Reply:
Usually there are no updater tests because they can be out of date with every update and potentially break especially when parsing html. Best to subscribe to another update channel like RSS or github releases and make sure through that that nothing is missed.